frontend/src/routes/Governance/policy-sets/components/PolicySetDetailSidebar.tsx

```diff
@@ -331,10 +331,10 @@ export function PolicySetDetailSidebar(props: { policySet: PolicySet }) {
         <Text component={TextVariants.p}>
           <Split hasGutter>
             <SplitItem>
-              <strong>{policySetClusters.length}</strong>&nbsp; clusters
+              <strong>{policySetClusters.length}</strong>&nbsp; {t('clusters')}
```

Comment:
Ideally this should use the <Trans> component with an interpolated value, rather than prefixing a translated string with a number. It should also handle plural/singular properly. (Same issue for policy count below.)